### PR TITLE
Feature/tree imageslider

### PR DESCRIPTION
**Pull request by Zoka-tech**

### CODE REVIEW

**Comment by doriengr** [comment]
Some of the files have missing empty lines :)

File: frontend/src/components/tree/Imageslider.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          >
+            <ArrowRight size={24} />
+          </button>
+        </div>
+      )}
+    </div>
+  )
+}
+
+export default Imageslider
```
Comment (doriengr) [inline frontend/src/components/tree/Imageslider.tsx:109]: `ImageSlider`. Please also rename the file 👍

**Review by doriengr — CHANGES_REQUESTED** [review]
It's really great to see that you created your own slider. The problem is that we already use a library for sliders called splide. 

Slider are kind of complicated. Expecially if you want to use transitions and different settings on different screen sizes. We also need to make sure, that the slider is accessible for people who can not see and use the website only with a screenreader. The slider library handles those cases for us. 

If you want to update your slider I can pinpoint the options that are missing. Otherwise I would suggest to use the slider library. 

Here is an example where the slider library is already in use: https://github.com/green-ecolution/green-ecolution-frontend/blob/develop/frontend/src/components/startpage/IntroductionSlider.tsx

File: frontend/src/components/tree/Imageslider.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    <div className="relative w-full">
+      <h2 className="font-bold text-xl mb-4">Hinterlegte Bilder:</h2>
+
+      <div className="flex space-x-4 mb-4 w-full">
+        {displayedImages.map((image, index) => (
+          <img
+            key={index}
+            src={image}
+            alt={`Slide ${currentIndex + index + 1}`}
+            className="w-full sm:w-1/3 md:w-1/4 h-48 sm:h-64 md:h-72 object-cover rounded-lg shadow-lg"
```
Comment (doriengr) [inline frontend/src/components/tree/Imageslider.tsx:71]: Please do not use a fixed height. Instead you can use aspect-ratio. That will insure, that the image has always the same ratio of width to height 👍

File: frontend/src/components/tree/ImageSlider.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+              <img
+                src={image}
+                alt={`Slide ${index + 1}`}
+                className="w-full aspect-square object-cover rounded-lg shadow-lg"
+              />
+          </SplideSlide>
+        ))}
+      </Splide>
+
+      {images.length > 1 && (
```
Comment (doriengr) [inline frontend/src/components/tree/ImageSlider.tsx:73]: Please check the [docs](https://splidejs.com/integration/react-splide/) of the slider. 

The navigation elements should also be used from the slider package itself.

File: frontend/src/components/tree/TreeDashboard.tsx [hunk 4]
```diff
@@ -115,4 +115,4 @@ const TreeDashboard = ({ treeId }: TreeDashboardProps) => {
   )
 }
 
-export default TreeDashboard
+export default TreeDashboard
```
Comment (doriengr) [inline frontend/src/components/tree/TreeDashboard.tsx:118]: Missing empty line

File: frontend/src/components/tree/TreeGeneralData.tsx [hunk 5]
```diff
@@ -52,7 +61,9 @@ const TreeGeneralData: React.FC<TreeGeneralData> = ({ tree }) => {
         ))}
       </dl>
 
-      <section className="mt-16">@TODO: Add image slider</section>
+      <section className="mt-16">
```
Comment (doriengr) [inline frontend/src/components/tree/TreeGeneralData.tsx:64]: `<ImageSlider images={images} />`

And put the margin on the wrapper of the slider

File: frontend/src/components/tree/TreeUpdate.tsx [hunk 6]
```diff
@@ -120,4 +120,4 @@ const TreeUpdate = ({ treeId, onUpdateSuccess, onUpdateError }: TreeUpdateProps)
   )
 }
 
-export default TreeUpdate
+export default TreeUpdate
```
Comment (doriengr) [inline frontend/src/components/tree/TreeUpdate.tsx:123]: Missing empty line

File: frontend/src/routes/_protected/tree/_formular/$treeId.edit.tsx [hunk 7]
```diff
@@ -62,4 +62,4 @@ function EditTreeCluster() {
       </Suspense>
     </div>
   )
-}
+}
```
Comment (doriengr) [inline frontend/src/routes/_protected/tree/_formular/$treeId.edit.tsx:65]: Missing empty line

**Review by doriengr — CHANGES_REQUESTED** [review]

File: frontend/src/components/tree/ImageSlider.tsx (hunk 3 above)
Comment (Zoka-tech) [reply frontend/src/components/tree/ImageSlider.tsx:73]: The navigation elements of splide look different than the one in Figma. Thats why I wrote that part. When I use the Splide arrows it looks like this:
Unbenannt

Is this okay?

**Comment by doriengr** [comment]
I am closing this PR because the implementation of images are on hold.

**Pull request closed without merging**